risc0/circuit/rv32im/README.md

```diff
@@ -1,2 +1,18 @@
 
-The RISC Zero zkVM circuit
\ No newline at end of file
+The RISC Zero zkVM circuit
+
+# Crate Feature Flags
+
+The following crate feature flags are available. They are configured in your `Cargo.toml`.
+
+* cuda
+  * Optional (default is CPU only)
+  * Turns on GPU acceleration for guest code with CUDA
+* metal
```

Comment:
For all `metal` flags it might be worth noting that it is only supported on `macos` OS's

Reply:
That's a really good point -- folks who don't use Mac might not recognize it as MetalTM. I wonder if we should also note support issues w/ CUDA on Mac or just leave it as an exercise for the reader since it's complicated.